suite-common/icons/src/icons.ts

```diff
@@ -111,6 +111,7 @@ export const cryptoIcons = {
     vtc: require('../assets/cryptoIcons/vtc.svg'),
     xrp: require('../assets/cryptoIcons/xrp.svg'),
     zec: require('../assets/cryptoIcons/zec.svg'),
+    sol: require('../assets/cryptoIcons/ada.svg'), // TODO(vl): add icons
```

Comment:
I think this is not gonna work because If you look into `README.md` of this package you should not change the file manually but run the generate script.

I found this when adding logo for Sepolia.

You can copy the `ada.svg` and name it `sol.svg` and it should generate it without any problem.

And I would do the same for `packages/components/src/components/assets/CoinLogo/coins.ts`. It's not required there but just for consistency.

Reply:
"first 5 commits are from previous PRs, so pls ignore"

This is from the part which is already merged. You are right that adding a new Icon which would be called `sol.svg` and then generating would be a better approach, but this worked too. This PR does not do anything with the icon, we have a separate PR prepared which adds Solana icon, if its OK, you will see that addressed there.